pages/test_analytics/js_collectors.md.erb

```diff
@@ -1,13 +1,14 @@
 # JavaScript Collectors
 
-Test Analytics ships with a Jest test collector.
+Test Analytics ships with JavaScript collectors :github: [`test-collector-javascript`](https://github.com/buildkite/test-collector-javascript).
+
 You can also upload test results by importing [JSON](/docs/test-analytics/importing-json) or [JUnit XML](/docs/test-analytics/importing-junit-xml).
 
 {:toc}
 
 ## Jest collector
 
-[Jest](https://jestjs.io/) is a JavaScript testing framework.
```

Comment:
If we're dropping the _describe what testing thing we're talking about_ language here, then perhaps we should do this for every framework (e.g., the RSpec docs has this same sort of text).

Perhaps this is worth saving for a separate PR. Really, we ought to have a template + checklist in the style guide for , which I'm happy to get started on, but the hard part is consensus.

Reply:
Let's offload that to a different PR, plz. You can create a newticket for this.

Reply:
(Though all other things being equal, I'd prefer to see the line change omitted as part of deferring the overall fix.)